app/src/main/java/org/maplibre/navigation/android/example/MockNavigationActivity.kt

```diff
@@ -35,6 +34,8 @@ import org.maplibre.turf.TurfConstants
 import org.maplibre.turf.TurfMeasurement
 import okhttp3.Request
 import org.maplibre.navigation.android.example.databinding.ActivityMockNavigationBinding
+import org.maplibre.navigation.android.navigation.ui.v5.route.NavigationMapRoute
```

Comment:
Aren't we introducing a dependency on the UI package by using classes from there? Maybe, instead of deleting the class here, we should move the newer/better implementation from the ui package to the core package?

Reply:
That not the core, this is the sample app.